### PR TITLE
feat(@kampus/pano): Initial folder setup with next

### PR DESCRIPTION
**Pull request by cansirin**
This PR introduces initial folder setup and a couple components related with the routing. 

Details:

-

### CODE REVIEW

File: apps/kampus/app/pano/features/post/PanoFilterLink.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+"use client";
+
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+
+export function PanoFilterLink({ href, children }: { href: string; children: React.ReactNode }) {
```
Comment (usirin) [inline apps/kampus/app/pano/features/post/PanoFilterLink.tsx]: let's convert this to a util & component that accepts the filter type (can be a string literal union), and returns an absolute url based on that.

**Review by usirin — CHANGES_REQUESTED** [review]
List of things we decided to do on stream:
- Use `/posts/....` for pages working with a list of posts
- Use `/post/....` for pages working with a single post
- Let's try to use query parameters for `/posts` pages. The idea is if we can use query parameters, we can utilize one page and let the filters get driven by query params. if we go with different routes, we lose the possibility to show "top rated pages from site.com"

File: apps/kampus/app/pano/features/post/PostSortFilters.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  { query: "all", label: "hepsi" },
+  { query: "active", label: "en günceller" },
+  { query: "hot", label: "en fazla yorum almışlar" },
+  { query: "liked", label: "en begenilenler" },
+  { query: "my-posts", label: "başlıklarım" },
+] as const;
+
+export const DEFAULT_FILTER_PATH = "/pano/posts?filter=all";
+
+export type PanoPostFilterType = (typeof filters)[number]["query"];
```
Comment (usirin) [inline apps/kampus/app/pano/features/post/PostSortFilters.tsx]: let's drive filters from the type we build, not the other way around.

File: apps/kampus/app/pano/features/filter/utils.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+export type PanoPostFilterType = "all" | "active" | "hot" | "liked" | "my-posts";
+
+export const filters: Record<PanoPostFilterType, string> = {
+  all: "hepsi",
+  active: "en son yorumlananlar",
+  hot: "en fazla yorum almışlar",
+  liked: "beğendiklerim",
+  "my-posts": "başlıklarım",
+};
```
Comment (Ketcap) [inline apps/kampus/app/pano/features/filter/utils.ts:9]: not sure about this but we can figure out later. It's interesting dictionary to work with.

**Review by Ketcap — APPROVED** [review]

**Review by usirin — APPROVED** [review]
💯

**Pull request merged**